src/rebar_paths.erl

```diff
@@ -0,0 +1,210 @@
+%% BEFORE THIS FIX: rebar3 ct  266.78s user 144.06s system 144% cpu 4:33.70 total
+%% CURRENT TIME:    rebar3 ct  419.30s user 301.00s system 152% cpu 7:51.98 total
+-module(rebar_paths).
+-include("rebar.hrl").
+
+-type target() :: deps | plugins.
+-type targets() :: [target(), ...].
```

Comment:
huh, don't think I've seen the use of `...` before. It is the same as `[target()]`, right? It isn't saying there can be types that aren't `target()` in there or anything like that?

Reply:
it's special notation to mean non-empty list containing `target()`